packages/validator/src/validator.ts

```diff
@@ -159,6 +162,24 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.getValidatorKeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        if (!indicesService.validatorPubKeyExists(pubkeyHex)) {
+          this.logger.info("Validator exists in beacon chain", {
+            validatorIndex: validatorStore.getIndexOfPubkey(pubkeyHex),
+            pubKey: pubkeyHex,
```

Comment:
this log is quite verbose and will most of the time not be a one-liner (depending on screen size) which makes is hard to read. I am wondering if we really need to print out the full pubkey here or just the first and last 4 bytes using `prettyBytes` https://github.com/ChainSafe/lodestar/blob/df1ef97e9050a588b74f21bfae314d675caea288/packages/utils/src/format.ts#L7

we already print out the full pubkey on when loading from keystore so it might be fine to not do it here

![image](https://user-images.githubusercontent.com/38436224/226317553-b7460b39-56bd-4b8e-ad9b-c6eb1be220e0.png)

Reply:
Good idea with `prettyBytes` that will be easier to read. The log is actually printed out by the [indicesService](https://github.com/ChainSafe/lodestar/blob/unstable/packages/validator/src/services/indices.ts#L138) which is ran per epoch, so I moved it here to separate concerns.